src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ProductTaxon.xml

```diff
@@ -15,6 +15,13 @@
                     <attribute name="groups">admin:product_taxon:read</attribute>
                 </attribute>
             </collectionOperation>
+            <collectionOperation name="admin_post">
```

Comment:
As you are adding new endpoints, add also contracts tests to cover it in `tests/Api/Admin` directory